packages/truffle-config/index.js

```diff
@@ -243,12 +252,32 @@ function Config(truffle_directory, working_directory, network) {
 };
 
 Config.prototype.addProp = function(key, obj) {
```

Comment:
This is already existing code, but I really dislike the variable names...I'm a little bit slow and when trying to read this, it destroys the narrative for me.  What about something like `function (propName, propValue)`?  What do you think?

Reply:
I also really dislike abbreviated variables like `str`.  That is just a subjective stylistic thing so feel free to tell me to go fly a kite.